website/pages/blog/introducing-the-slack-source-plugin.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Introducing the Slack Source Plugin
+tag: announcement
+date: 2022/12/20
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Slack source plugin.
+author: hermanschaaf
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are happy to announce the release of the Slack source plugin for CloudQuery.
```

Comment:
Can we use the same tagline as on the website?

Reply:
@yevgenypats "CloudQuery is an open source high performance data integration platform designed for security and infrastructure teams." ?

Reply:
Updated